examples/02_meta-analyses/plot_cbma_subtraction_conjunction.py

```diff
@@ -0,0 +1,128 @@
+from nimare.tests.utils import get_test_data_path
+
+###############################################################################
+# Load Sleuth text files into Datasets
+# --------------------------------------------------
+# The data for this example are a subset of studies from a meta-analysis on
+# semantic cognition in children. [3]_ A first group of studies probed
+# childrens semantic world knowledge (e.g., correctly naming an object after
+# hearing its auditory description) while a second group of studies asked
+# children to decide if two (or more) words were semantically related to one
+# another or not.
+sleuth_file = os.path.join(get_test_data_path(), "semantic_knowledge_children.txt")
+sleuth_file2 = os.path.join(get_test_data_path(), "semantic_relatedness_children.txt")
+
+dset = convert_sleuth_to_dataset(sleuth_file)
+dset2 = convert_sleuth_to_dataset(sleuth_file2)
+
+###############################################################################
+# Individual group ALEs
+# --------------------------------------------------
+# Computing seperate ALE analyses for each group is not strictly necessary for
+# performing the subtraction analysis but will help to appreciate the
+# similarities and differences between the groups.
+ale = ALE(null_method="approximate")
+res = ale.fit(dset)
+res2 = ale.fit(dset2)
+
+corr = FWECorrector(method="montecarlo", voxel_thresh=0.001, n_iters=1000)
+cres = corr.transform(res)
+cres2 = corr.transform(res2)
+
+img = cres.get_map("z_level-cluster_corr-FWE_method-montecarlo")
+plot_stat_map(
+    img,
+    cut_coords=4,
+    display_mode="z",
+    title="Semantic knowledge",
+    threshold=2.326,  # cluster-level p < .01, one-tailed
+    cmap="RdBu_r",
+    vmax=4,
+)
+
+img2 = cres2.get_map("z_level-cluster_corr-FWE_method-montecarlo")
+plot_stat_map(
+    img2,
+    cut_coords=4,
+    display_mode="z",
+    title="Semantic relatedness",
+    threshold=2.326,  # cluster-level p < .01, one-tailed
+    cmap="RdBu_r",
+    vmax=4,
+)
+
+###############################################################################
+# Subtraction analysis
+# --------------------------------------------------
+sub = ALESubtraction(n_iters=1000, low_memory=False)
+res_sub = sub.fit(dset, dset2)
+img_sub = res_sub.get_map("z_desc-group1MinusGroup2")
+
+plot_stat_map(
+    img_sub,
+    cut_coords=4,
+    display_mode="z",
+    title="Subtraction",
+    threshold=2.576,  # voxel-level p < .01, two-tailed
+    cmap="RdBu_r",
+    vmax=4,
+)
+
+###############################################################################
+# Conjunction analysis
+# --------------------------------------------------
+# To determine the overlap of the meta-analytic results, a conjunction image
+# can be computed by (a) identifying voxels that were statistically significant
+# in *both* individual group maps and (b) selecting, for each of these voxels,
+# the smaller of the two group-specific *z* values. [4]_
```

Comment:
Could you also mention that, since this is simple image math, there is no NiMARE implementation for conjunction analysis, but it can be done easily with nilearn?